src/main/scala/mesosphere/marathon/core/election/ExponentialBackoff.scala

```diff
@@ -0,0 +1,27 @@
+package mesosphere.marathon.core.election
+
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.duration._
+
+private class ExponentialBackoff(
```

Comment:
We have this functionality elsewhere (e.g. RateLimiter).
Perhaps we can reuse logic?